reference/system-databases/cluster-log.md

```diff
@@ -0,0 +1,73 @@
+---
+title: CLUSTER_LOG
+summary: Learn the `CLUSTER_LOG` cluster log table.
+category: reference
+---
+
+# CLUSTER_LOG
+
+You can query cluster logs on the `CLUSTER_LOG` cluster log table. By pushing down query conditions to each node, the impact of the query on cluster performance is less than that of the `grep` command.
```

Comment:
@reafans Please help confirm whether it's `instance` or `node` here and other places in this file.
```suggestion
You can query cluster logs on the `CLUSTER_LOG` cluster log table. By pushing down query conditions to each instance, the impact of the query on cluster performance is less than that of the `grep` command.
```